util/dvsim/dvsim.py

```diff
@@ -411,19 +416,19 @@ def parse_args():
 
     buildg.add_argument("--build-opts",
                         "-bo",
-                        nargs="+",
-                        default=[],
-                        metavar="OPT",
+                        default='',
+                        metavar="OPTS",
                         help=('Additional options passed on the command line '
```

Comment:
other helps mention comma separated lists where appropriate, should this one?

Reply:
This isn't comma separated: it's treated as something that should be added to the command (split up with `shlex.split()` on line 637). I spent a while trying to think of a good way to say that and failed! Can you think of a pithy description?